src/cljs/commiteth/ui_model.cljs

```diff
@@ -92,16 +102,22 @@
     ::bounty-filter-type.predicate                     (fn [filter-value bounty]
                                                          (->> filter-value
                                                               (some #{(:repo-owner bounty)})
-                                                              boolean))}})
+                                                              boolean))}
+
+   ::bounty-filter-type|claims
+   {::bounty-filter-type.name      "Claims"
+    ::bounty-filter-type.category  ::bounty-filter-type-category|single-static-option
+    ::bounty-filter-type.options   bounty-filter-type-claims-options-def
+    ::bounty-filter-type.predicate (fn [filter-value bounty]
+                                     (condp = filter-value
+                                       ::bounty-filter-type-claims-option|no-claims
+                                       (<= 0 (:claim-count bounty))))}})
```

Comment:
Should be `(= 0 (:claim-count bounty))`.

Reply:
Just changed! Actually I did that on purpose to make the code more resilient, so for example `(= 0 nil)` is false but `(<= 0 nil)` is true. But it is probably rather cryptic, so I just left the equality as you mentioned and we will see if we have any more needs.

Reply:
Understand now, did not about that ClojureScript (or rather Javascript!) trick. In that case it should have been `>=`, not `<=`. But yes, it's rather cryptic, `(= 0 (or (:claim-count bounty) 0))` or `(-> bounty :claim-count some? not)` is longer but clearer. Clojure is good:)